packages/core/admin/admin/src/components/AuthenticatedApp/index.js

```diff
@@ -31,6 +31,7 @@ const AuthenticatedApp = () => {
   const userInfo = auth.getUserInfo();
   const userName = get(userInfo, 'username') || getFullName(userInfo.firstname, userInfo.lastname);
   const [userDisplayName, setUserDisplayName] = useState(userName);
+  const [userId, setUserId] = useState('');
```

Comment:
use not use null as default value instead of an empty string ?

Reply:
Used empty string because we have an empty string everywhere, where events are anonymous (or malfunctioning for some reason), so just kept consistency. Another small reason is that it indicates it should be a string. But it can be null if that's better for code consistency across the code base

Reply:
Yeah overall if there is no value you would rather want to use a null|undefined to represent "no-value" rather than an empty string. Do you need to convert it to a string when sending the events ?

Reply:
Nope, Amplitude will be fine with null. I will change it 👍